app/src/main/java/org/thoughtcrime/securesms/onboarding/pickname/PickDisplayNameActivity.kt

```diff
@@ -54,9 +57,11 @@ class PickDisplayNameActivity : BaseActionBarActivity() {
 
         setComposeContent { DisplayNameScreen(viewModel) }
 
+        if (!failedToLoad) prefs.setHasViewedSeed(false)
```

Comment:
What is the logic for here? Does `setHasViewedSeed` still makes sense in the new onboarding logic?
What 'seed' has been viewed?